apps/ui/src/components/ProposalResults.vue

```diff
@@ -22,26 +22,27 @@ const labels = {
   2: 'Abstain'
 };
 
-const progress = computed(() => Math.min(props.proposal.scores_total / props.proposal.quorum, 1));
+const totalProgress = computed(() =>
+  Math.min(props.proposal.scores_total / props.proposal.quorum, 1)
+);
 
-const adjustedScores = computed(() =>
-  props.proposal.scores.map(score => {
-    // TODO: sx-api returns number, sx-subgraph returns string
-    const parsedTotal = parseFloat(props.proposal.scores_total as unknown as string);
+const results = computed(() => {
+  // TODO: sx-api returns number, sx-subgraph returns string
+  const parsedTotal = parseFloat(props.proposal.scores_total as unknown as string);
 
-    return parsedTotal !== 0 ? (score / parsedTotal) * 100 * progress.value : 0;
-  })
-);
+  return props.proposal.scores
+    .map((score, i) => {
+      const progress = parsedTotal !== 0 ? (score / parsedTotal) * 100 : 0;
```

Comment:
I can't figure out why it was multiplied by progress before, I somewhat recall that we made this adjustment because of some weird quorum logic, but looking at the (new) code it looks correct.

Do you have any insight @bonustrack?

If we can't recall anything I think we can merge it and see if it causes some issues

Reply:
It was introduced here https://github.com/snapshot-labs/sx-monorepo/pull/228.

I assumed that the `results` variable was holding the correct progress and score from its name, did not know that everything was relative to the quorum

Reply:
I think that's we wanted initially, that this bar with results shows quorum left:
<img width="257" alt="image" src="https://github.com/snapshot-labs/sx-monorepo/assets/1968722/0df08733-2b5b-4b0b-8666-2d64295b9b62">

Without this it would never show progress towards quorum.